tools/cvm-image-rewriter/tdx-libvirt-ubuntu-host.xml.template

```diff
@@ -1,10 +1,10 @@
 <domain type='kvm' xmlns:qemu='http://libvirt.org/schemas/domain/qemu/1.0'>
-  <name>td-guest-grub-on-ubuntu-host</name>
-  <memory unit='KiB'>2097152</memory>
+  <name>tdx-guest</name>
```

Comment:
How about adding a random string or changing the name to a variable for creating multiple TDs?

Reply:
The script provides arg "-n" to set domain name.